### DISCUSSION TITLE
Installation Error

### DISCUSSION THREAD

**Discussion by yoddaa** [shun126/UE5-DungeonGeneratorDemo#6] [Q&A] [answered]
![Screenshot (301)](https://github.com/shun126/UE5-DungeonGeneratorDemo/assets/49987506/8b80cc6c-4790-4ab3-a59e-e38dc211cf78)

after restart 
cannot find 2 folders and closing unreal5

how can I solve it?

thank you :)

**Comment by shun126** [comment]
Really sorry for the delay in getting back to you.
Is the situation the same after updating to the latest plugin?
Also, what version of the Unreal Engine are you using?

**Reply by yoddaa** [reply]
Is the situation the same after updating to the latest plugin? version 1.5
Also, what version of the Unreal Engine are you using? UE 5.3.1

thx

**Reply by shun126** [reply, marked as answer]
Does installing version 1.5.9 of the plugin not improve the situation?
